src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html

```diff
@@ -148,8 +148,9 @@ <h3>JJ decision</h3>
           <p class="section-grid-header">&nbsp;</p>
           <p class="section-jj-grid-text">
             <input formControlName="revisedDueDate" type="date" [hidden]="!(timeToPay=='yes')"
-              style="background-color:#e4f2ff; width: 100%; font-size: smaller; border: 0"
+              style="background-color:#e4f2ff; width: 100%; font-size: smaller; border: 0" (change)="showDateHint = false"
               [value]="jjDisputedCount?.revisedDueDate">
+            <mat-hint style="font-size:x-small">{{ jjDisputedCount?.revisedDueDate && showDateHint && timeToPay =='yes' ? jjDisputedCount?.revisedDueDate.substring(0,10) : "" }}</mat-hint>
```

Comment:
added to show the value coming from DB for revisedDueDate since this is one the JJ can change.  had to use a non-material UX on front end just a plain HTML input type=date which angular does not want to initialize on binding with a form control.  On entry to this form the hint will show the current value and user can type in the input.  When they start typing, the hint will disappear since the input will have the new value.